README.md

```
@@ -1,137 +1,445 @@
-|`helpers`|`translators.Grandfather`|❌|✅|✅|✅|Create translations based on existing translated resources.
-|`helpers`|`translators.Repetitions`|❌|✅|✅|✅|Create translations based on leverage of 100% text matches.
-|`helpers`|`translators.Visicode`|❌|✅|✅|✅|Pseudo-localization with visual identification of string id's.
-|`helpers-deepl`|`DeepL`|✅|❌|✅|💰|DeepL translation [API](https://www.deepl.com/docs-api).
-|`helpers-demo`|`PigLatinizer`|✅|❌|✅|✅|Translator into pig latin for demo and pseudo-localization.
-|`helpers-googlecloud`|`GoogleCloudTranslateV3`|✅|❌|✅|💰|Google Translate V3 [API](https://cloud.google.com/translate/docs).
-|`helpers-translated`|`ModernMT`|✅|✅|✅|💰|Modern MT translation [API](https://www.modernmt.com/api/#introduction) (both realtime and batch).
-|`helpers-translated`|`TranslationOS`|✅|❌|✅|✅|TOS human translation [API](https://api.translated.com/v2).
-|`helpers-xliff`|`BridgeTranslator`|✅|❌|✅|❌|Translator via XLIFF files in filesystem.
-
-## Other
+|**Internal Providers**|||||
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 7; Actual: 5; Too few cells, row will be missing data

(MD056, table-column-count)